README.rst

```diff
@@ -123,6 +123,23 @@ PUT JSON data::
              (lambda (&key data &allow-other-keys)
                (message "I sent: %S" (assoc-default 'json data)))))
 
+Another PUT JSON example (nested JSON using alist structure, how to represent a boolean & how to selectively evaluate lisp)::
+
+  (request
+   "http://httpbin.org/put"
+   :type "PUT"
+    :data (json-encode `(("jsonArray" . (        ; Prepend alist structure with a backtick (`) rather than single quote (') to allow elisp evaluation of selected elements prefixed with a comma (,)
```

Comment:
Indentation is wrong. And is this comment necessary ? Emacs Lisp programmers understand this thing so I think this comment is redundant.